committee-steering/sig-governance.md

```diff
@@ -0,0 +1,465 @@
+  - Setup *sessions* for Kubecon
+- Facilitate Kubecon contributor summit topics
+  - At least 1 admin from each SIG should be invited to the contributor summit
+- Setup programs for the SIG such as Outreachy
+  - Manage communication with any programs external to the SIG
+- Either author or facilitate authorship of release notes for
+  changes made to areas owned by the SIG
+
+#### Selecting Administrators
+
+SIG ~~Foo~~ maintains 2-3 administrators.  When an administrator steps
+down from their position, they may propose a new
+administrator from the SIG ~~Foo~~ members to take their place.  Lazy
+consensus should follow.  If lazy consensus cannot be achieved
+within the SIG, then by default the proposal is upheld and the motion
+passes.
+
+If deemed necessary by the members of SIG ~~Foo~~, the number of
+administrators may be expanded.  This may be proposed by SIG members
+and determined by a super majority.
+
+Administrators that have not been active for a period of 8 weeks or more
+(exceptions exist) should step down and propose a replacement.  For
+planned absences, Administrators should find a temporary replacement
+to fill their role and delegate responsibility.
+
+When possible, it is preferred that the Admin group represent a
+diverse set of organizations.  This is predicated on interest in
+the role from multiple organizations with qualified candidates.
+
+### Technical Lead role
+
+Canonical list of admins: kubernetes-sig-~~foo~~-techleads@googlegroups.com
+
+The SIG ~~Foo~~ technical leads group is responsible for resolving
+technical disputes and setting technical direction for the SIG.
+Technical leads should be well versed in most technical areas
+governed by the SIG and have shown consistent excellent technical
+judgement and leadership over a sustained period.  Technical leads
+should be focused on directing others within the SIG and are
+ultimately responsible for the technical direction.
+
+Technical leads may appear in OWNERs files as approvers, but not
+every member who is an approver is a technical lead for the SIG.
+
+Technical leads may also be Administrators, however the membership
+and responsibilities of each role is tracked separately.
+
+Technical leads that have held the position for a year and step down may
+retain the title "SIG ~~Foo~~ administrator emeritus".
+
+#### Internally facing responsibilities
+
+- Act as role model within the SIG
+  - Other members will look up to technical leads in the SIG.  Leads
+    must model respectful behavior towards all members of the community
+- Act as an escalation point for technical disputes within the SIG
+- Ensure technical excellence within area
+  - Actively working to reduce complexity of the area
+  - Defining general acceptance criteria for work
+    - code style guide
+    - go doc style guide
+    - test requirements
+    - documentation requirements
+  - Make technical decisions either with other technical leads
```

Comment:
Should we also add a requirement about attending SIG architecture to discuss technical changes with the wider community?

Reply:
Also added a bit about working cross organizationally when appropriate